src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsUapHiViFandomMobile.java

```diff
@@ -0,0 +1,285 @@
+    TimeUnit.SECONDS.sleep(3);
+    hiViUap.clickVideo();
+
+    Assert.assertTrue(startProgressBarWidth < hiViUap.getProgressBarWidth(), "Video time indicator should move.");
+  }
+
+  @Test
+  public void shouldMuteVideoForAutoplayedImpactState() throws InterruptedException {
+    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+    fandomPage.waitForPageLoad();
+    HiViUap hiViUap = new HiViUap(driver, TLB_SLOT_ID);
+    hiViUap.waitForVideoStart();
+
+    TimeUnit.SECONDS.sleep(3);
+    Assertion.assertFalse(SoundMonitor.wasSoundHeardOnPage(new JavascriptActions()));
+    TimeUnit.SECONDS.sleep(3);
+    Assertion.assertFalse(SoundMonitor.wasSoundHeardOnPage(new JavascriptActions()));
+  }
+
+  @Test
+  public void shouldUnmuteVideoAfterClickOnIcon() throws InterruptedException {
+    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+    fandomPage.waitForPageLoad();
+    HiViUap hiViUap = new HiViUap(driver, TLB_SLOT_ID);
+
+    hiViUap.waitForVideoStart();
+    hiViUap.clickVideo();
+    hiViUap.toggleSound();
+    TimeUnit.SECONDS.sleep(3);
+
+    Assertion.assertTrue(SoundMonitor.wasSoundHeardOnPage(new JavascriptActions()));
+  }
+
+  @Test
+  public void shouldPlayUnmutedVideoForReplayedAd() throws InterruptedException {
+    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+    fandomPage.waitForPageLoad();
+    HiViUap hiViUap = new HiViUap(driver, TLB_SLOT_ID);
+
+    hiViUap.waitForVideoEnd();
+    hiViUap.clickReplayButton();
+    TimeUnit.SECONDS.sleep(3);
+
+    Assertion.assertTrue(SoundMonitor.wasSoundHeardOnPage(new JavascriptActions()));
+  }
+
+  @Test
+  public void shouldRedirectToPageAfterClickOnAd() {
+    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+    fandomPage.waitForPageLoad();
+    fandomPage.refreshPage();
+    fandomPage.waitForPageLoad();
+    HiViUap hiViUap = new HiViUap(driver, TLB_SLOT_ID);
+    hiViUap.clickVideo();
+    hiViUap.clickAd();
+    fandomPage.waitForPageLoad();
+
+    Assert.assertTrue(fandomPage.tabContainsUrl(AD_REDIRECT));
+  }
+
+  @Test
+  public void shouldRedirectAfterClickOnLearnMore() {
+    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+    fandomPage.waitForPageLoad();
+    HiViUap hiViUap = new HiViUap(driver, TLB_SLOT_ID);
+    hiViUap.clickLearnMore();
+    fandomPage.waitForPageLoad();
+
+    Assert.assertTrue(fandomPage.tabContainsUrl(AD_REDIRECT));
+
+  }
+
+  @Test
+  public void shouldPauseOnVideoAfterClickOnPauseIcon() throws InterruptedException {
+    AdsFandomObject fandomPage = loadArticle(FandomAdsDataProvider.PAGE_HIVI_UAP_ARTICLE);
+    fandomPage.waitForPageLoad();
+    HiViUap hiViUap = new HiViUap(driver, TLB_SLOT_ID);
+
+    hiViUap.waitForVideoStart();
+    TimeUnit.SECONDS.sleep(2);
+    hiViUap.clickVideo(); // to trigger ui
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)